Symfony/CS/Fixer/Symfony/SpacesCastFixer.php

```diff
@@ -23,6 +23,14 @@ class SpacesCastFixer extends AbstractFixer
     /**
      * {@inheritdoc}
      */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound(array(T_ARRAY_CAST, T_BOOL_CAST, T_DOUBLE_CAST, T_INT_CAST, T_OBJECT_CAST, T_STRING_CAST, T_UNSET_CAST));
```

Comment:
We have `Token::isCast`: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/495d7e6502d9fc38c40660683ff276ac61057ce2/Symfony/CS/Tokenizer/Token.php#L287

Maybe we should add a method `getCastTokenKinds`, like `getClassyTokenKinds`?